### ISSUE TITLE
No download/progress bar in vimperator

### ISSUE THREAD

**Issue by GiedriusS** [vimperator/vimperator-labs#3]
When you use vimperator with gui=none there is no way of knowing how much % of a web page is loaded and there is no way of knowing how much time/size left until a download finishes. This is a pretty big issue and I'm sure it annoys not just me. It should show some simple basic information about this in the bottom bar (look @ how dwb does it). 
Workaround for seeing downloads information is to open another window with the download list (:em down<tab><enter>) and for web page loading - no idea.

**Comment by danemacmillan** [comment]
@Roger, this is completely unrelated to the ticket, but how did you get that padlock icon next to the URL?

**Comment by jamalis** [comment]
Gee thanks a lot, with the post above I became rather well-informed about the discussions.
http://fotorumahmewah.com

**Comment by gkatsev** [comment]
I agree that having a progress bar of sorts would be nice but it's unlikely to happen any time soon.

**Comment by kesdoputr** [comment]
Maybe a late question,how can i put download button(or other button)
on the status bar? LIke @Roger's picture above.

**Comment by timss** [comment]
@kesdoputr Use `:dialog customizetoolbar`, and drag the download button to the statusline.

**Comment by kesdoputr** [comment]
Wow,i don't think for this method,i just think this is some plugin.
Thanks for your reply.
